### PR TITLE
(MAINT) add docs re: tk auth and disks on perf hardware

### PR DESCRIPTION
**Pull request by cprice404**

### CODE REVIEW

**Comment by fpringvaldsen** [comment]
:see_no_evil:

**Comment by cprice404** [comment]
@nwolfe would you mind taking a quick gander at this

File: README_tk_auth.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            sort-order: 1
+            name: "allow all"
+        },
+```
+
+After this, it's a good idea to do an agent run with `puppet agent -t` to verify that PE doesn't think it needs to modify the `auth.conf` file.
+
+After verifying that, you will want to restart or HUP the server so that the new rules take effect.  You can test the rules by running a command like this:
+
+   curl -k https://localhost:8140/puppet/v3/catalog/<certname-of-this-server>\?environment=production
```
Comment (nwolfe) [inline README_tk_auth.md]: This isn't rendering correctly in Markdown. I think the `<certname-of-this-server>` problematic?  We can probably just change that part to `$(hostname)` like we have in other parts of the documentation.

**Comment by nwolfe** [comment]
:+1:

**Comment by cprice404** [comment]
@nwolfe added a commit to fix that, thx

**Comment by nwolfe** [comment]
Hm, not sure why it's not rendering properly still.  Tabs vs spaces?  Not enough spaces?  Oh well, merging

**Comment by cprice404** [comment]
seems to render ok for me?  who knows.  computers--

**Pull request merged**